qiita_ware/commands.py

```diff
@@ -89,26 +92,31 @@ def submit_EBI(preprocessed_data_id, action, send, fastq_dir_fp=None):
                         ("Submitting XMLs for pre_processed_id: "
                          "%d" % preprocessed_data_id))
         try:
-            # place holder for moi call see #1477
-            xmls_cmds_moi = xmls_cmds
-        except:
-            # handle exception
-            LogEntry.create('Fatal', seqs_cmds,
-                            info={'ebi_submission': preprocessed_data_id})
+            xml_content, stderr, _ = system_call(xmls_cmds)
+
+        except Exception as e:
+            error = str(e)
+            LogEntry.create('Fatal', error,
+                            info={'ebi_submission': preprocessed_data_id,
+                                  'fail': cmd})
         else:
             LogEntry.create('Runtime',
                             ('Submission of sequences of pre_processed_id: '
                              '%d completed successfully' %
                              preprocessed_data_id))
+        finally:
```

Comment:
If `system_call` raises, `xml_content` is not defined, and `stderr` doesn't contain the value from such call (it will have the value from the above call to `system_call` in the loop.